canonical-web-and-design/jaas-dashboard
# Render the action logs output to the Actions Logs page

## Done

- Added button in unit list to view Action Logs.
- Added tab in Model details to view Action Logs.
- Requested the action logs from the API.
- Display the data in an unstyled table.

## QA

- Ask me for access to the demo model.
- View the demo.
- Click to view the action logs tab in the top menu, wait, see the logs.
- View the unit list, click to view the action logs, wait, see the logs.

## Details

Fixes: https://github.com/canonical-web-and-design/juju-squad/issues/1666
Fixes: https://github.com/canonical-web-and-design/juju-squad/issues/1668
Fixes: https://github.com/canonical-web-and-design/juju-squad/issues/1669

## Screenshots
![Screen Shot 2021-04-19 at 2 46 49 PM](https://user-images.githubusercontent.com/532033/115301304-3c701000-a11e-11eb-9599-ea6e6a6706aa.png)
